src/component-base.mjs

```diff
@@ -33,17 +33,17 @@ export default class ComponentBase extends LitElement {
     /**
      * @private
      */
-    this.ariaPressed = 'false';
+    this.ariapressed = 'false';
```

Comment:
`ariaPressedState(ariaPressed)` function should reflect this change

Reply:
Maybe the one used in that function should be renamed. It is not related to the class property `this.ariapressed`

Reply:
The function arg is not really even used in the function it is pass into. It just gets passed in and returned right back out without an mutations. Going to make a issue to explore a refactor later. 

```
ariaPressedState(ariaPressed) {
    const ariaToggle = function (event) {
      const ariaPressedNode = this.shadowRoot.querySelector('[aria-pressed]');
      ariaPressedNode.setAttribute("aria-pressed", 'false');

      if (event.type === 'mousedown') {
        ariaPressedNode.ariaPressed = true;
      } else {
        ariaPressedNode.ariaPressed = false;
      }

      if (event.type === 'keydown') {
        if (event.code === 'Enter' || event.code === 'Space') {
          ariaPressedNode.ariaPressed = true;
        } else {
          ariaPressedNode.ariaPressed = false;
        }
      }
    };

    // Add event listeners
    this.addEventListener('mousedown', ariaToggle);
    this.addEventListener('mouseup', ariaToggle);
    this.addEventListener('keydown', ariaToggle);
    this.addEventListener('keyup', ariaToggle);

    return ariaPressed;
  }
```